src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import Navbar from './components/Navbar';
+import Rockets from './components/Rockets';
+import Missions from './components/Missions';
+import Profile from './components/Profiles';
+import { fetchRocket } from './redux/rockets/rockets';
+
+function App() {
```

Comment:
- [ ] Test cases are missing for this project! Kindly ensure to add unit tests with React Testing Library and Jest (for unit tests).